reference/format/assets.rst

```diff
@@ -1,2 +1,25 @@
 Assets
 ======
+
++-----------+----------------+
+| Filename  | ``assets.yml`` |
++-----------+----------------+
+| Root Node | ``css``        |
++-----------+----------------+
+
+The root ``css`` node is used to structure your CSS files into logical groups:
```

Comment:
Please add more description what groups are. More info you can find here https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/AsseticBundle/README.md